src/script/DeployLineaWorldID.s.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import { Script } from "forge-std/src/Script.sol";
+import { LineaWorldID } from "../LineaWorldID.sol";
+
+contract DeployLineaWorldID is Script {
+    LineaWorldID public lineaWorldId;
+
+    address public messageServiceAddress;
+    address public remoteSenderAddress;
+
+    ///////////////////////////////////////////////////////////////////
+    ///                            CONFIG                           ///
+    ///////////////////////////////////////////////////////////////////
+    string public root = vm.projectRoot();
+    string public path = "src/script/.deploy-config.json";
+    string public json = vm.readFile(path);
+
+    uint8 public treeDepth;
+
+    function setUp() public {
+        // LineaWorldID is not deployed yet, so use the deployer's address as a placeholder
+        address tempAddress = msg.sender;
+
+        remoteSenderAddress = tempAddress;
```

Comment:
We don't need it because `Ownable` constructor automatically sets the owner of `LineaWorldID` contract to `msg.sender` (and `CrossDomainOwnable`'s `isLocal` is true by default).
```suggestion
```